main.js

```diff
@@ -0,0 +1,120 @@
+/* When the user clicks on the button,
+toggle between hiding and showing the dropdown content */
+function dropdownFunction(id) {
+    let idString = "myDropdown" + id;
```

Comment:
se schimba vreodata idString in aceasta functie?

Reply:
da, se schimba penbtru fiecare intrare din 'tabel' in functie de ce id are cryptomoneda in db